File: server-aspnet-fastendpoints/src/Org.OpenAPITools/Models/ParticipantResponse.cs
namespace Org.OpenAPITools.Models;


/// <summary>
/// 
/// </summary>
public class ParticipantResponse 
{
    public long Id { get; set; }
    public AccountMiniResponse Account { get; set; }
    public long AvailableStart { get; set; }
    public long AvailableEnd { get; set; }
    public string Color1 { get; set; }
    public string Color2 { get; set; }
    public string Type { get; set; }
}

File: server-scala-play-server/app/model/ParticipantResponse.scala
package model

import play.api.libs.json._

/**
  * Represents the Swagger definition for ParticipantResponse.
  */
@javax.annotation.Generated(value = Array("org.openapitools.codegen.languages.ScalaPlayFrameworkServerCodegen"), date = "2026-02-09T20:57:39.376804970Z[Etc/UTC]", comments = "Generator version: 7.20.0-SNAPSHOT")
case class ParticipantResponse(
  id: Option[Long],
  account: Option[AccountMiniResponse],
  availableStart: Option[Long],
  availableEnd: Option[Long],
  color1: Option[String],
  color2: Option[String],
  `type`: Option[String]
)

object ParticipantResponse {
  implicit lazy val participantResponseJsonFormat: Format[ParticipantResponse] = Json.format[ParticipantResponse]
}

